src/html/partials/behind-the-scenes.html

```diff
@@ -1,4 +1,57 @@
 <section class="behind-scenes-section">
+
   <h2 class="sm-title">Behind The Scenes</h2>
-  <p class="bg-intro">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
-</section>
+  <p class="lg-text">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
+
+  <!-- <div class="behind-scenes-section__florence">
+    <div class="behind-scenes-section__florence__book"></div>
+    <div class="behind-scenes-section__florence__gear"></div>
+    <div class="behind-scenes-section__florence__comment"></div>
+    <div class="behind-scenes-section__florence__heart"></div>
+    <div class="behind-scenes-section__florence__link"></div>
+  </div> -->
+
+  <ul class="behind-scenes-section__list">
+    <li class="behind-scenes-section__list__item">
+      <button class="behind-scenes-section__list__item__btn--book"></button>
+      <div class="behind-scenes-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Medical Knowledge Bases</h3>
+        <p class="sm-sans-serif-text">Side effects and drug interactions are currently available in services like <a href="https://www.epocrates.com/" target="_blank">Epocrates</a> and <a href="https://www.medscape.com/" target="_blank">Medscape</a>. Though designed for long-form reading, it could be pruned and adapted to fit messaging constraints.</p>
+      </div>
+    </li>
+
+    <li class="behind-scenes-section__list__item">
+      <button class="behind-scenes-section__list__item__btn--gear"></button>
+      <div class="behind-scenes-section__list__item__wrapper">
```

Comment:
i know this is just a one pager and not at risk for naming issues but say you wanted something that looked like this but it wasn't in a section named "Behind the Scenes" - what would you name this? i think that unhinging the class name from the specific content is the best way to approach class naming, because if someone came onto this project after you, saw a style like this used on something that wasn't a "behind the scenes" section, they would be pretty confused and unsure if they were using the class name correctly. maybe something like `plotted-point` or `pinned-note` would make more sense?

Reply:
Yes, @prechter, this gets to one of the other modified approaches to BEM which I neglected to talk with you about. Will chat with you more about it later this morning.